frontend/src/state/schema.js

```diff
@@ -34,6 +34,19 @@ export const layer = new schema.Entity(
   {},
   {
     processStrategy: (l) => {
+      const getTimeline = () => {
+        const parseDate = timeParse(l.attributes.timeline_format);
```

Comment:
Correct me if I'm wrong but I think this code is incorrect. `timeline_format` stores the format used to _display_ the date in the UI. It does not store the format in which the date is stored in the database. This means that you should only use `timeline_format` when rendering a `Date` object in the UI.

In this file, I would only convert what's coming from the API to a `Date` object and store `timeline_format` for when it's needed in the components.

You're using the same approach in `frontend/src/state/modules/layers/utils.js` and I think it is also incorrect.

For example, `timeline_format` may have a value of `'%Y'` but the API will still provide you `timeline_default_date` as a `%Y-%m-%d` string.